sympy/physics/quantum/matrixutils.py

```diff
@@ -3,7 +3,6 @@
 from __future__ import print_function, division
 
 from sympy import MatrixBase, I, Expr, Integer
-from sympy.core.compatibility import range
```

Comment:
I assume that this would give xrange instead of range under Python 2.

Reply:
Maybe. `range` is not used in the file so that was the main motivation to remove it.

Reply:
Okay!